### DISCUSSION TITLE
Configuration lost after restart

### DISCUSSION THREAD

**Discussion by sysqo82** [sct/overseerr#3091] [Help] [answered]
Good morning,
I have Overseerr installed as a docker image on a windows 10 pc.
I have used the recommended intall command but every time I restart my pc, I need to re-login and re-config both radarr and sonarr servers, the rest of the setup (requests, movies, series etc.) is left untouched, but the task of doing it all over again is tedious.
I read that I need to have a setup volume under my docker setup, but there isn't ANY explanation on how to create it.
Thank you.
![volumes](https://user-images.githubusercontent.com/20111075/197327493-40a3c0eb-daaa-42bf-9958-e928960ecc35.png)
![containers](https://user-images.githubusercontent.com/20111075/197327496-5c7d0051-4559-45c6-a41b-d06f096c9d92.png)
![images](https://user-images.githubusercontent.com/20111075/197327497-0d32e6c1-9569-4f66-8014-b70d0052b6f0.png)

**Comment by TheCatLady** [comment, marked as answer]
https://docs.overseerr.dev/getting-started/installation#windows

**Reply by sysqo82** [reply]
Thank you, I have no idea how I missed this when I read it.
